planet/spider.py

```diff
@@ -235,6 +243,36 @@ def writeCache(feed_uri, feed_info, data):
           if os.path.exists(cache_file): os.remove(cache_file)
           continue
 
+        if config.fedora_messaging_enabled() and not os.path.exists(cache_file):
+          # If the cache file for this entry doesn't exist, then we can
+          # somewhat safely assume we have never seen it before and that
+          # it is "new"
+          try:
+            msg = fm_publish(
+              PostNew(
+                topic="planet.post.new",
+                body=dict(
```

Comment:
Per @nphilipp's comment in another PR ``dict()`` is significantly slower than using ``{}``, should we change?